lib/src/boollist.dart

```diff
@@ -42,18 +42,18 @@ abstract class BoolList with ListMixin<bool> {
   factory BoolList(int length, {bool fill = false, bool growable = false}) {
     RangeError.checkNotNegative(length, 'length');
 
-    BoolList boolist;
+    BoolList boolList;
```

Comment:
careful about unrelated changes

Reply:
I'm very careful ... to not forget to fix them.